docs/guide/exe_verilator_coverage.rst

```diff
@@ -154,10 +158,6 @@ generated from random test runs) into one master coverage file.
 Specifies the aggregate coverage results, summed across all the files,
 should be written to the given filename in :command:`lcov` .info format.
 This may be used to feed into :command:`lcov` to aggregate or generate
-reports.
-
-Converting from the Verilator coverage data format to the info format is
-lossy; the info will have all forms of coverage merged line coverage, and
-if there are multiple coverage points on a single line they will merge.
```

Comment:
I still think this multiple points is true when user-defined "cover" statements are used.